assets/js/components/ClusterDetails/ClusterSettings.jsx

```diff
@@ -51,54 +40,83 @@ export function ClusterSettings({ usingNewChecksEngine = false }) {
-                  'focus:outline-none',
-                  selected
-                    ? 'bg-white shadow'
-                    : 'text-gray-800 hover:bg-white/[0.12]'
-                )
-              }
-            >
-              {tabTitle}
-            </Tab>
-          ))}
-        </Tab.List>
-        <ClusterInfoBox haScenario={cluster.type} provider={cluster.provider} />
-        {cluster.provider === UNKNOWN_PROVIDER && (
-          <WarningBanner>
-            The following catalog is valid for on-premise bare metal platforms.
-            <br />
-            If you are running your HANA cluster on a different platform, please
-            use results with caution
-          </WarningBanner>
-        )}
-        <Tab.Panels className="mt-2">
-          {Object.entries(tabsSettings).map(([tabTitle, tabContent]) => (
-            <Tab.Panel
-              key={tabTitle}
-              className={classNames(
-                'bg-white rounded p-3',
-                'focus:outline-none focus:ring-2 ring-offset-2 ring-white ring-opacity-60'
-              )}
-            >
-              {tabContent}
-            </Tab.Panel>
-          ))}
-        </Tab.Panels>
-      </Tab.Group>
+      {usingNewChecksEngine ? (
+        <ChecksSelectionNew clusterId={clusterID} cluster={cluster} />
+      ) : (
+        <RunnerSettings clusterID={clusterID} cluster={cluster} />
+      )}
+    </div>
+  );
+}
+
+function RunnerSettings({ clusterID, cluster }) {
+  const tabsSettings = {
+    'Checks Selection': (
+      <ChecksSelection clusterId={clusterID} cluster={cluster} />
+    ),
+    'Connection Settings': (
+      <ConnectionSettings clusterId={clusterID} cluster={cluster} />
+    ),
+  };
+
+  return (
+    <Tab.Group manual>
+      <Tab.List className="flex p-1 space-x-1 bg-zinc-300/20 rounded">
+        {Object.keys(tabsSettings).map((tabTitle) => (
+          <Tab
+            key={tabTitle}
+            className={({ selected }) =>
+              classNames(
+                'w-full py-2.5 text-sm leading-5 font-medium rounded',
+                'focus:outline-none',
+                selected
+                  ? 'bg-white shadow'
+                  : 'text-gray-800 hover:bg-white/[0.12]'
+              )
+            }
+          >
+            {tabTitle}
+          </Tab>
+        ))}
+      </Tab.List>
+      <InfoBox haScenario={cluster.type} provider={cluster.provider} />
+      <Tab.Panels className="mt-2">
+        {Object.entries(tabsSettings).map(([tabTitle, tabContent]) => (
+          <Tab.Panel
+            key={tabTitle}
+            className={classNames(
+              'bg-white rounded p-3',
+              'focus:outline-none focus:ring-2 ring-offset-2 ring-white ring-opacity-60'
+            )}
+          >
+            {tabContent}
+          </Tab.Panel>
+        ))}
+      </Tab.Panels>
+    </Tab.Group>
+  );
+}
+
+export function InfoBox({ haScenario, provider }) {
```

Comment:
I wouldn't create a new exported `InfoBox` function in a component that will be deleted soonish.
I would replicate this if needed in the `ChecksSelectionNew` file, so later on the removal is seamless